snuba/migrations/snuba_migrations/stacktraces/0001_stacktraces.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Column,
+    DateTime,
+    String,
+    UInt,
+    NamedTuple,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
```

Comment:
`elapsed_relative_to_start_date_ns`, `thread_id` and `queue_name` are going to be in the JSON object stored in the `stacktrace` column. We chose not to have several rows for each entry (as we were doing previously) since it's simpler to integrate into the Sentry SDKs that way.